frontend/src/components/registration-form/hooks/caregiver.form.js

```diff
@@ -0,0 +1,49 @@
+export const caregiverFormFields = [
+  //General information
+  [
+    { id: "first_name", type: "text", name: "first_name", placeholder: "Unesite Vaše ime*" },
+    { id: "last_name", type: "text", name: "last_name", placeholder: "Unesite Vaše prezime*" },
+    { id: "email", type: "email", name: "email", placeholder: "Unesite Vaš E-mail*" },
+    { id: "phone_number", type: "text", name: "phone_number", placeholder: "Unesite Vaš broj telefona*" },
+    { id: "password", type: "text", name: "password", placeholder: "Unesite Vašu lozinku*" },
+    {
+      id: "postal_code",
+      type: "number",
+      name: "postal_code",
+      placeholder: "Poštanski broj*",
+    },
+    { id: "city", type: "text", name: "city", placeholder: "Mesto stanovanja*" },
+    { id: "birthdate", type: "datepicker", name: "birthdate", placeholder: "Dan / Mesec / Godina rodjenja*" },
+    {
+      id: "gender",
+      type: "dropdown",
+      name: "gender",
+      placeholder: "Pol*",
+      options: ["muški", "ženski"],
+    },
+  ],
+  //Caregiver fields
+  [
+    {
+      id: "work_application",
+      type: "dropdown",
+      name: "work_application",
+      placeholder: "Prijava za rad*",
+      options: ["prijava-rad"],
+    },
+    { id: "experience", type: "text", name: "experience", placeholder: "Iskustvo*" },
+    {
+      id: "weekly_days",
+      type: "dropdown",
+      name: "weekly_days",
+      placeholder: "Dostupnost*",
+      options: [1, 2, 3, 4, 5, 6, 7],

```

Comment:
Ovi `options`, ako se budu koristili u `Select` komponenti, bi trebalo da bacaju error jer `Select` komponenta ocekuje object za `options` a ne array. 
Jesi li testirao ovo?
Takodje, ima vise mesta na kojima je `options` array a ne objekat.

Reply:
Da, svuda je ostalo array, ne baca gresku ali vidim sad da ocekuje objekat. Vraticu to, ali hajde isto da se cujemo za svaki input da odlucimo ako je select koje opcije ce da ima

Reply:
Vazi... Ako ti treba da listu pretvoris u objekat koji ce biti prihvatljiv u `Select` komponenti kao `options`, mozes da uradis:
`const options = optionsArray.reduce((previous, current) => ({ ...previous, [current]: current }), {});`

Time ces od `[1,2,3]` dobiti `{1:1, 2:2, 3:3}`.